daml-script/daml3/Daml/Script/Questions/PartyManagement.daml

```diff
@@ -15,8 +15,10 @@ instance IsQuestion AllocateParty Party where command = "AllocateParty"
 -- | A hint to the backing participant what party id to allocate.
 -- Must be a valid PartyIdString (as described in @value.proto@).
 newtype PartyIdHint = PartyIdHint { partyIdHint : Text }
+  deriving (Ord, Eq, Show)
```

Comment:
Naive question. Why do we add that ?

Reply:
Ah, oops! I added it for some code I ended up removing. I'm tempted to leave it in though, as I feel like we should have had these from the start

Reply:
I'll remove these and add a ticket to go through all the daml-script exposed data types and give them reasonable instances

Reply:
[#17724](https://github.com/digital-asset/daml/issues/17724)